packages/office-addin-dev-certs/scripts/uninstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+hashes=$(security find-certificate -c "$1" -a -Z | grep SHA-1 | awk '{ print $NF }')
```

Comment:
Will this change also work on Windows? I recall the reason we never completed the original change for this issue is that the solution I wrote up wouldn't work cross-platform?

Another open issue I recall is that the original solution would not remove ALL expired certs, but rather only the first one it encountered.  Will this remove all expired certs?

Reply:
This is for bash script and uses the mac "security" command and I don't know if that works for other platforms or not (seems like it should for linux, but I don't have a test environment for that).  Right now windows uses powershell script files.

It does remove all the certificates (expired or otherwise) . . . that's why the for loop below.  Note I think the powershell for windows deletes them all as well.

The discussion I saw in you change (https://github.com/OfficeDev/Office-Addin-Scripts/pull/245/files) centered around the deletion being a side effect of checking if installed and the duplication of code.  I avoided those things by beefing up the uninstall and verify commands for mac to be more accurate instead.